tests/test_utils.py

```diff
@@ -271,27 +270,31 @@ class PlainRelatedResourceTypeSerializer(serializers.Serializer):
     assert get_related_resource_type(field) == output
 
 
-def test_get_included_serializers():
-    class IncludedSerializersModel(DJAModel):
-        self = models.ForeignKey("self", on_delete=models.CASCADE)
-        target = models.ForeignKey(ManyToManyTarget, on_delete=models.CASCADE)
-        other_target = models.ForeignKey(ManyToManyTarget, on_delete=models.CASCADE)
+class IncludedSerializersModel(DJAModel):
+    self = models.ForeignKey("self", on_delete=models.CASCADE)
+    target = models.ForeignKey(ManyToManyTarget, on_delete=models.CASCADE)
+    other_target = models.ForeignKey(ManyToManyTarget, on_delete=models.CASCADE)
 
-        class Meta:
-            app_label = "tests"
+    class Meta:
+        app_label = "tests"
 
-    class IncludedSerializersSerializer(serializers.ModelSerializer):
-        included_serializers = {
-            "self": "self",
-            "target": ManyToManyTargetSerializer,
-            "other_target": "tests.serializers.ManyToManyTargetSerializer",
-        }
 
-        class Meta:
-            model = IncludedSerializersModel
-            fields = ("self", "other_target", "target")
+class IncludedSerializersSerializer(serializers.ModelSerializer):
+    included_serializers = {
+        "self": "self",
+        "target": ManyToManyTargetSerializer,
+        "other_target": "tests.serializers.ManyToManyTargetSerializer",
+    }
+
+    class Meta:
+        model = IncludedSerializersModel
+        fields = ("self", "other_target", "target")
 
-    included_serializers = get_included_serializers(IncludedSerializersSerializer)
+
+def test_get_included_serializers():
+    included_serializers = getattr(
```

Comment:
From an API perspective this is a bit ugly that `getattr` always needs to be called instead of simply `serializer.included_serializers`. I would prefer that `included_serializers` is set to `None` when not configured so it easier to read `included_serializers` for a DJA user.

Reply:
This comment has not been addressed yet. Easiest I guess is to set `included_serializers` in the serializer base class to `None` so most of those `getattr` can be removed and simply property opened.